lbrynet/daemon/Publisher.py

```diff
@@ -54,6 +55,42 @@ def create_and_publish_stream(self, name, bid, claim_dict, file_path, claim_addr
     @defer.inlineCallbacks
     def publish_stream(self, name, bid, claim_dict, claim_address=None, change_address=None):
         """Make a claim without creating a lbry file"""
+        log.info('Starting publish for %s', name)
+
+        source = claim_dict['stream']['source'].get('source', None)
+        source_type = claim_dict['stream']['source'].get('sourceType', None)
+        content_type = claim_dict['stream']['source'].get('contentType', 'application/octet-stream')
```

Comment:
There is no file_path though, that's the prerequisite for this function running in the first place